tests/fx/optimization/test_transformations.py

```diff
@@ -211,59 +221,189 @@ def reverse(self, graph_module):
                 self.assertFalse(t1.transformed(node))
 
 
-def test_merge_linears():
-    _, traced = get_bert_model()
-    transformation = MergeLinears()
-    transformed = transformation(traced)
+class CustomTransformationsTests(unittest.TestCase):
```

Comment:
Otherwise the tests are not discovered in https://github.com/huggingface/optimum/blob/69f2883d945ccc669bdd0c78e0ba8fe4265c6792/.github/workflows/test_fx.yml#L37

By the way, I just moved `test_transformations.py` to `tests/fx`, otherwise is it not discovered either.

Reply:
We should change that to:
```
python -m pytest tests/fx
```

That way we dont have to create unnecessary classes, and we can keep the directory tree.

Reply:
@michaelbenayoun I rolled back to having a subfolder. I use `python -m pytest fx/optimization/test_transformations.py --exitfirst` in the yml because `test_quantization.py` is currently not run on main, and the tests don't pass; independenty from this PR (`python -m unittest discover -s fx -p 'test_*.py'` was not capturing it, see e.g. https://github.com/huggingface/optimum/actions/runs/3068109805/jobs/4955165866 )